src/components/Login/Login.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Login.css';
+import { Link } from 'react-router-dom';
+
+export default function Login() {
+  return (
+    <>
+    <div className="login-container">
+        <h3 className="login-heading">Account Information</h3>
```

Comment:
I don't see this line on the screenshot, is this required?

Reply:
Done ,with the changes